src/lang/en/settings.json

```diff
@@ -14,9 +14,16 @@
   "favicon": "Favicon",
   "filename_char_mapping": "Filename char mapping",
   "forward_direct_link_params": "Forward direct link params",
-  "github_client_id": "Github client id",
-  "github_client_secrets": "Github client secrets",
-  "github_login_enabled": "Github login enabled",
+  "sso_client_id": "Single sign-on client id",
```

Comment:
This file is auto generated, so you don't need to edit it manually

Reply:
> This file is auto generated, so you don't need to edit it manually

all right, i edited it just for better look while debugging